src/app/commands/new_params.cpp

```diff
@@ -55,6 +56,17 @@ void Param<std::string>::fromString(const std::string& value)
   setValue(value);
 }
 
+template<>
+void Param<doc::algorithm::ResizeMethod>::fromString(const std::string& value)
+{
+  if (value == "bilinear" || value == "Bilinear")
```

Comment:
Instead of asking for `bilinear` and `Bilinear`, use `base::utf8_icmp(a, b) == 0`, check `Param<doc::ColorMode>::fromString` for an example.

Reply:
Got it! A new function in my mind.